app/main.jsx

```diff
@@ -25,9 +26,10 @@ const ExampleApp = connect(
 render(
   <Provider store={store}>
     <Router history={browserHistory}>
-      <Route path="/" component={Products}>
-        <IndexRedirect to="/products" />
+      <Route path="/navbar" component={Navbar} />
```

Comment:
Here you're setting up a front end path to render your Navbar and making other components children of that route so that you are always rendering it. But that means that the user sees or needs to type in "www.blah.com/navbar/whatever" for the page to work. That's not ideal. So, I would make a main App component that renders all your permanent components and set that to be the main front end route. It can then render its children routes within there. Look to Juke for an example of this!